yarn.lock

```diff
@@ -15243,22 +15742,7 @@ __metadata:
   languageName: node
   linkType: hard
 
-"chai@npm:^4.3.10, chai@npm:^4.3.7":
-  version: 4.5.0
-  resolution: "chai@npm:4.5.0"
-  dependencies:
-    assertion-error: "npm:^1.1.0"
-    check-error: "npm:^1.0.3"
-    deep-eql: "npm:^4.1.3"
-    get-func-name: "npm:^2.0.2"
-    loupe: "npm:^2.3.6"
-    pathval: "npm:^1.1.1"
-    type-detect: "npm:^4.1.0"
-  checksum: cde341aee15b0a51559c7cfc20788dcfb4d586a498cfb93b937bb568fd45c777b73b1461274be6092b6bf868adb4e3a63f3fec13c89f7d8fb194f84c6fa42d5f
-  languageName: node
-  linkType: hard
-
-"chai@npm:^4.3.4, chai@npm:^4.3.6":
+"chai@npm:4.5.0, chai@npm:^4.3.4":
```

Comment:
@ltyu I believe this line is the source of the error you're seeing in CI. A 4.5.0 version of chain shouldn't be resolving to 4.3.6, which has a lower minor version. Did yarn do this automatically? Let's consider simply upgrading chai across the monorepo. It's better to use consistent versions anyway

Reply:
Fixed. The issue was that i manually changed all the package.json references. the correct way to do it is with `yarn up chai@4.5.0`